# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/arrayvec/ci/miri.sh
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/arrayvec/ci/miri.sh b/third_party/rust/arrayvec/ci/miri.sh
old mode 100644
new mode 100755
--- a/third_party/rust/arrayvec/ci/miri.sh
+++ b/third_party/rust/arrayvec/ci/miri.sh
@@ -1,9 +1,9 @@
-#!/usr/bin/env sh
+#!/bin/sh
 
 set -ex
 
 export CARGO_NET_RETRY=5
 export CARGO_NET_TIMEOUT=10
 
 MIRI_NIGHTLY=nightly-$(curl -s https://rust-lang.github.io/rustup-components-history/x86_64-unknown-linux-gnu/miri)
 echo "Installing latest nightly with Miri: $MIRI_NIGHTLY"